observatory_platform/telescopes/fundref.py

```diff
@@ -508,7 +510,7 @@ class FundrefTelescope:
 
     DAG_ID = 'fundref'
     DESCRIPTION = 'The Funder Registry dataset: https://www.crossref.org/services/funder-registry/'
-    DATASET_ID = DAG_ID
+    DATASET_ID = 'crossref'
```

Comment:
I think it might actually be more correct if it was called crossref_funder_registry. I just read a little deeper, and it used to be called fundref, but is now called funder registry.